src/components/NewLocationPage/NotesBlock/NotesBlock.tsx

```diff
@@ -6,22 +6,45 @@ import {
   IconWrapper,
   TextContainer,
   TextComponent,
+  StyledBaseButton,
 } from './NotesBlock.style';
+import { EventAction, EventCategory } from 'components/Analytics';
 
-const NotesBlock: React.FC<{ icon: any; title: string }> = ({
+interface NotesProps {
+  icon: any;
+  title: string;
+  redirectUrl?: string;
+  trackingCategory?: EventCategory;
+}
+
+const NotesBlock: React.FC<NotesProps> = ({
   icon,
   title,
+  redirectUrl,
+  trackingCategory = EventCategory.NONE,
```

Comment:
Also curious why we have a default of NONE here?  Would it be better to require it to always be specified?  or is NONE actually a good default?

Reply:
hmm, maybe a generic location page header category as a backup? if the note were clickable, there _should_ be an event category passed in. `NONE` was just a fallback for if the note is ever just text and not clickable (i dont know if this would ever be the case?) 

im gonna leave it as `NONE` for now and revisit when i go over analytics for the new components tomorrow

Reply:
Oh, gotcha.  Another option would be to do something like

```
assert(trackingCategory || !redirectUrl, 'Must supply a trackingCategory if this is a link.')
```

but cool to revisit later. 👍